### PR TITLE
feat: new logo suggestions

### PR DESCRIPTION
**Pull request by aburakt**
added some recommendations for logo

### CODE REVIEW

**Comment by aburakt** [comment]
we have a logo now 🥳

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-<center><img src='./assets/icon.svg' height="100"/>
+<center><img src='./assets/logo/logo.svg' height="100"/>
```
Comment (ufukbakan) [inline README.md:1]: i think ```<center>``` is not working in github markdown, 
can you change it to ```<h2 align="center">```

Comment (ufukbakan) [reply README.md:1]: also height="100" looks small for this logo

**Comment by ufukbakan** [comment]
LGTM

**Pull request merged**